src/gui/src/gui.cpp

```diff
@@ -421,6 +421,11 @@ Renderer::~Renderer()
   gui::Gui::get()->unregisterRenderer(this);
 }
 
+void Renderer::redraw()
+{
+  Gui::get()->redraw();
```

Comment:
Does this trigger a fullRepaint?

Reply:
Yes, I modified the connection in mainWindow.cpp from just `update()` to `fullRepaint()` so calling that should force the cache to be cleared and repainted on the next paintevent